tests/unit/sftp_test.py

```diff
@@ -48,6 +60,22 @@ def dst_path(file_paths_pair):
     return path
 
 
+@pytest.fixture
+def other_payload():
+    """Generate a binary test payload."""
+    uuid_name = uuid.uuid4()
+    return 'Original content: {name!s}'.format(name=uuid_name).encode()
+
+
+@pytest.fixture
+def dst_exists_path(file_paths_pair, other_payload):
+    """Return a data destination path."""
+    path = file_paths_pair[1]
+    path.write_bytes(other_payload)
```

Comment:
Why can't we just use `transmit_payload` here?
```suggestion
def dst_exists_path(file_paths_pair, transmit_payload):
    """Return a data destination path."""
    path = file_paths_pair[1]
    path.write_bytes(transmit_payload)
```

Also, I'd rename this fixture to `pre_existing_dst_path` to communicate the intent better.

Reply:
Because we want to be sure that the existing file has different content than the transmitted one to make sure it will be correctly overridden with the new data. Renaming is a good idea though.

Reply:
ack